src/Chapter1.hs

```diff
@@ -429,6 +429,7 @@ task is to specify the type of this function.
 49
 -}
 
+squareSum :: Integral a => a -> a -> a
```

Comment:
We explain typeclasses in the following chapters, so you are one step ahead 😸

In here `Int -> Int -> Int` would also work, as a more specific type 🙂

And `Num` instead of `Integral` would work as well, as a less restrictive typeclass.